### ISSUE TITLE
Text contrast is too low on LandingPage.

### ISSUE THREAD

**Issue by Roger-Heathcote** [fac19/prepare-me#59]
White on turquoise does not have height enough contrast ratio. It is currently 2.5, it should be 3 for large text and 4.5 for small.
Poss alternatives: black on white? black on yellow?
